src/NSwag.SwaggerGeneration.AspNetCore.Tests/AspNetCoreToSwaggerGenerationTests.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System.Linq;
```

Comment:
Should this have a license header?

Reply:
The other tests didn't - https://github.com/RSuter/NSwag/blob/master/src/NSwag.SwaggerGeneration.WebApi.Tests/ControllerClassesTests.cs. @RSuter ?

Reply:
I was too lazy to add it in the tests :)

Reply:
Cool. I'll go and add it to new tests as I go through them.